### PR TITLE
removed composer --dev option everywhere

### PR DESCRIPTION
**Pull request by fabpot**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | n/a |
| License | MIT |
| Doc PR | n/a |

### CODE REVIEW

File: src/Symfony/Bridge/ProxyManager/README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Resources
 You can run the unit tests with the following command:
 
     $ cd path/to/Symfony/Bridge/ProxyManager/
-    $ composer.phar install --dev
+    $ composer.phar install
```
Comment (xabbuh) [inline src/Symfony/Bridge/ProxyManager/README.md]: What do you think about removing the `.phar` extension here and below for consistency?

Comment (fabpot) [inline src/Symfony/Bridge/ProxyManager/README.md]: At least, the .phar is consistent everywhere, but yeah, that makes sense; let's do it.

File: src/Symfony/Component/Intl/Resources/bin/update-data.php [hunk 2]
```diff
@@ -50,7 +50,7 @@
 For running this script, the intl extension must be loaded and all vendors
 must have been installed through composer:
 
-composer install --dev
+composer install
```
Comment (WouterJ) [inline src/Symfony/Component/Intl/Resources/bin/update-data.php]: for documentation consistency, what about prefixing this with `$`?

````
For running this script, the intl extension must be loaded and all vendors
must have been installed through composer:

    $ composer install
````

**Comment by Tobion** [comment]
:+1:

**Pull request merged**